opencensus/common/backports/__init__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opencensus.common.backports.backports import WeakMethod
+
+
+__all__ = [
+    'WeakMethod',
+]
```

Comment:
I'm worried about these alias-style init modules getting out of sync, and it'll be tedious updating these with each change to the code. 

What about moving the code from e.g. the old `backports.py` into `backports/__init__.py` instead? This is still pretty ugly -- lots of code in `__init__.py` files -- but it fixes the stutter and solves the problem of keeping code in sync.

Reply:
Either way it is ugly as you said.
I'm fine with moving code to `__init__.py` directly, will do it now.